.eslintrc.json

```diff
@@ -0,0 +1,37 @@
+{
+  "env": {
+    "browser": true,
+    "jest": true,
+    "node": true,
+    "webextensions": true,
+    "es2020": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:react/recommended",
+    "plugin:@typescript-eslint/eslint-recommended",
+    "plugin:@typescript-eslint/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 11,
+    "sourceType": "module"
+  },
+  "settings": {
+    "react": {
+      "createClass": "createReactClass", // Regex for Component Factory to use,
+      // default to "createReactClass"
+      "pragma": "React", // Pragma to use, default to "React"
+      "version": "detect" // React version. "detect" automatically picks the version you have installed.
+    }
+  },
+  "plugins": ["react", "@typescript-eslint"],
+  "rules": {
+    "@typescript-eslint/no-explicit-any": "off",
+    "react/prop-types": ["off"],
+    "no-console": "warn"
```

Comment:
I'll trust you on these rules as long as most of them were ported over from tslint.

Reply:
Well I didn't want to mess around too much with the new defaults. So the first two,

```
"@typescript-eslint/no-explicit-any": "off",
"react/prop-types": ["off"],
```
will eventually be remove once we refactor some more, but this is a good start for now.